.github/workflows/ci.yaml

```diff
@@ -17,4 +18,5 @@ jobs:
         with:
           node-version: ${{ matrix.node-version }}
       - run: yarn install
-      - run: yarn test
+      - run: yarn test:cjs
+      - run: [[ "${{ matrix.node-version }}" = 18 ]] && yarn test:esm || true
```

Comment:
Does this work? I thought you need `if` like this:

https://github.com/vercel/nft/blob/cc7761a63e188b2b26567ba524e5edd573c0eefa/.github/workflows/ci.yml#L42